BlackJack/index.js

```diff
@@ -29,189 +41,295 @@ makeCard();
-playerCards.appendChild(playerImg)
-
-let playerRandomCard2 = Math.floor(Math.random() * allCard.length) +1;
-playerImg2.src = `cards/${allCard[playerRandomCard2]}`
-playerCards.appendChild(playerImg2)
+playerImg.src = `cards/${allCard[playerRandomCard]}`;
+playerCards.appendChild(playerImg);
 
+let playerRandomCard2 = Math.floor(Math.random() * allCard.length) + 1;
+playerImg2.src = `cards/${allCard[playerRandomCard2]}`;
+playerCards.appendChild(playerImg2);
 
 //save player first card value
-let firstCardN = allCard[playerRandomCard]
-let secondCardN = allCard[playerRandomCard2]
-let firstCardNum = firstCardN.charAt(0)
-let secondCardNum = secondCardN.charAt(0)
+let firstCardN = allCard[playerRandomCard];
+let secondCardN = allCard[playerRandomCard2];
+let firstCardNum = firstCardN.charAt(0);
+let secondCardNum = secondCardN.charAt(0);
 
-if(firstCardNum + secondCardNum === '1A' || firstCardNum + secondCardNum === 'JA' || firstCardNum + secondCardNum === 'QA' || firstCardNum + secondCardNum === 'KA' ){
-  alert('BlackJack You Win')
+if (
+  firstCardNum + secondCardNum === "1A" ||
+  firstCardNum + secondCardNum === "JA" ||
+  firstCardNum + secondCardNum === "QA" ||
+  firstCardNum + secondCardNum === "KA"
+) {
+  alert("BlackJack You Win");
   location.reload();
-}else if(firstCardNum + secondCardNum === 'A1' || firstCardNum + secondCardNum === 'AJ' || firstCardNum + secondCardNum === 'AQ' || firstCardNum + secondCardNum === 'AQ' ){
-  alert('BlackJack You Win')
+} else if (
+  firstCardNum + secondCardNum === "A1" ||
+  firstCardNum + secondCardNum === "AJ" ||
+  firstCardNum + secondCardNum === "AQ" ||
+  firstCardNum + secondCardNum === "AQ"
+) {
+  alert("BlackJack You Win");
   location.reload();
 }
 
-function firstCardNumFunc (){
-  let firstCardNum = firstCardN.charAt(0)
-  if(firstCardNum === '2' || firstCardNum === '3' || firstCardNum === '4' || firstCardNum === '5' || firstCardNum === '6' || firstCardNum === '7' || firstCardNum === '8' || firstCardNum === '9'){
-     return firstCardN = firstCardNum;
-  }else if( firstCardNum === 'J' || firstCardNum === 'K' || firstCardNum === 'Q' || firstCardNum === '1'){
-    return firstCardN = '10'
-  }else if(firstCardNum === 'A'){
-  return firstCardN = '1'
+function firstCardNumFunc() {
+  let firstCardNum = firstCardN.charAt(0);
+  if (
+    firstCardNum === "2" ||
+    firstCardNum === "3" ||
+    firstCardNum === "4" ||
+    firstCardNum === "5" ||
+    firstCardNum === "6" ||
+    firstCardNum === "7" ||
+    firstCardNum === "8" ||
+    firstCardNum === "9"
+  ) {
+    return (firstCardN = firstCardNum);
+  } else if (
+    firstCardNum === "J" ||
+    firstCardNum === "K" ||
+    firstCardNum === "Q" ||
+    firstCardNum === "1"
+  ) {
+    return (firstCardN = "10");
+  } else if (firstCardNum === "A") {
+    return (firstCardN = "1");
   }
 }
-firstCardNumFunc ()
-console.log(firstCardN)
+firstCardNumFunc();
+console.log(firstCardN);
```

Comment:
Good attempt, I can understand what you're trying to achieve here. However we should avoid changing the data type that is being stored in global variables. Global variables mean variables that are available throughout the whole file.
firstCardN is storing the card filename itself, meaning 9-D.png as example. later on it is assigned with the string "10". This causes confusion when the code base is large. You ideally want to keep separate variables to store separate things.